commands/command_filter_process.go

```diff
@@ -73,6 +72,12 @@ func filterCommand(cmd *cobra.Command, args []string) {
 		ExitWithError(err)
 	}
 
+	skip := filterSmudgeSkip
```

Comment:
I think you could simplify this to:

```go
skip := filterSmudgeSkip || cfg.Os.Bool("GIT_LFS_SKIP_SMUDGE", false)
```

which I find to be more readable in context.